gulpfile.js

```diff
@@ -28,15 +29,17 @@ const globsToBeautify = [
 
 // Task implementations
 function beautify() {
-  return gulp.src(globsToBeautify, {
+  gulp.src(globsToBeautify, {
       base: './'
     })
-    .pipe(prettify())
+    .pipe(prettify({
+      eol: os.EOL, // \r\n on Windows, \n on POSIX
```

Comment:
@nmatkins, I tested that this works fine on Windows, @ZakMiller, let me know if it gives you any issues on Mac